mandatory/2-function-creation.js

```diff
@@ -16,17 +19,42 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
-
+function formatPercentage(num) {
+  //let numb = num.toFixed(2);
+  //return num > 100 ? "100%" : `${num.toFixed(2)}%`;
+  return num > 100 ? "100%" : `${Math.round(num * 100) / 100}%`;
+}
+console.log(formatPercentage(99.372));
+console.log(Math.round(99.372 * 100) / 100);
+console.log(Math.round(0.372 * 100) / 100);
 /*
 Write a function that:
 - takes an array of strings as input
 - removes any spaces in the beginning or end of each string
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  console.log(arrayOfStrings);
+  let newArr = arrayOfStrings.map((string) => {
+    //return string.toLowerCase();
+    return string.replace(/^(\s+)?(\/+)?|(\s+)?(\/+)?(\s+)?$/g, "").toLowerCase();
+  });
```

Comment:
you can also use `.trim` to remove the leading and trailing whitespace (you're not expected to know all the standard functions in JS, of course, but this one was mentioned in the week1 course). Then you can deal with the `/` removal